catalog/ngis-cat/projects_ngis-cat_assets_DEA_NIDEM.jsonnet

```diff
@@ -0,0 +1,179 @@
+    with a 30-year time series archive of spatially and spectrally calibrated Landsat satellite data managed within the Digital Earth Australia (DEA) platform. 
+    NIDEM complements existing intertidal extent products, and provides data to support a new suite of use cases that require a more detailed understanding 
+    of the three-dimensional topography of the intertidal zone, such as hydrodynamic modelling, coastal risk management and ecological habitat mapping.
+    The product: https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#basics
+    The definition of the four bands, including NIDEM, NIDEM_mask, NIDEM_uncertainty, and NIDEM_unfiltered please visit https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#details
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'australia',
+    'dea_australia',
+    'dem',
+    'nidem',
+  ],
+  providers: [
+    ee.producer_provider('Digital Earth Australia', 'https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#basics'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+
+  extent: ee.extent(108.81, -44.41, 157.82, -9.13, '1986-08-16T00:00:00Z', '2017-07-31T23:59:59Z'),
+
+  // Summaries contain additional information specific to the dataset type.
+  summaries: {
+
+    'gee:schema': [
+      {
+        name: 'dea_national_intertidal_digital_elevation_model',
+        description: |||
+          The National Intertidal Digital Elevation Model (NIDEM; Bishop-Taylor et al. 2018, 2019) is a continental-scale elevation dataset for Australia's exposed intertidal zone. 
+          NIDEM provides the first three-dimensional representation of Australia's intertidal sandy beaches and shores, tidal flats and rocky shores and reefs at 25 m spatial resolution, 
+          addressing a key gap between the availability of sub-tidal bathymetry and terrestrial elevation data. NIDEM was generated by combining global tidal modelling 
+          with a 30-year time series archive of spatially and spectrally calibrated Landsat satellite data managed within the Digital Earth Australia (DEA) platform. 
+          NIDEM complements existing intertidal extent products, and provides data to support a new suite of use cases that require a more detailed understanding 
+          of the three-dimensional topography of the intertidal zone, such as hydrodynamic modelling, coastal risk management and ecological habitat mapping.
+          The product: https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#basics
+          The definition of the four bands, including NIDEM, NIDEM_mask, NIDEM_uncertainty, and NIDEM_unfiltered please visit https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#details
+        |||,
+
+        type: ee_const.var_type.double,
+      },
+    ],
+    platform: [
+      'DEA',
+    ],
+    instruments: [
+      'DEA NIDEM',

```

Comment:
What physical instrument is this referring to?

I need to better document platform and instrument. Can you describe how the data was created so I can try to help figure out the platform and instrument or if we should drop the fields

Reply:
This is a developed product, so the `instruments` field is removed.